apps/condo/domains/meter/constants/constants.js

```diff
@@ -36,6 +37,7 @@ const METER_READING_SOURCE_MESSENGER_TYPE = 'messenger'
 const METER_READING_SOURCE_SOCIAL_NETWORK_TYPE = 'social_network'
 const METER_READING_SOURCE_MOBILE_APP_STAFF_TYPE = 'mobile_app_staff'
 const METER_READING_SOURCE_CRM_TYPE = 'crm'
+const METER_READING_SOURCE_EXTERNAL_INTEGRATION_TYPE = 'external_integration'
```

Comment:
How will you get what external integration created the reading? `sender` and `updatedBy`?

Reply:
> `sender` and `updatedBy`

Yes